app/Models/Teacher.php

```diff
@@ -15,16 +15,16 @@ class Teacher extends Model
 
     public function user()
     {
-    	return $this->belongsTo(User::class,'id');
+        return $this->belongsTo(User::class,'id');
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Generic.Functions.FunctionCallArgumentSpacing.NoSpaceAfterComma</br>**Severity**: error</br>**File**: app/Models/Teacher.php</br>**Line**: 18</br>No space found after comma in function call